Looms.PoS/Looms.PoS.Application/Features/Order/Commands/CreateOrder/CreateOrdersCommandHandler.cs

```diff
@@ -0,0 +1,46 @@
+using Looms.PoS.Application.Interfaces;
+using Looms.PoS.Application.Interfaces.ModelsResolvers;
+using Looms.PoS.Application.Models.Requests;
+using Looms.PoS.Domain.Interfaces;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Looms.PoS.Application.Features.Order.Commands.CreateOrder;
+
+public class CreateOrdersCommandHandler : IRequestHandler<CreateOrdersCommand, IActionResult>
+{
+    private readonly IOrdersRepository _ordersRepository;
+    private readonly IBusinessesRepository _businessRepository;
+    private readonly IUsersRepository _usersRepository;
+    private readonly IHttpContentResolver _httpContentResolver;
+    private readonly IOrderModelsResolver _modelsResolver;
+
+    public CreateOrdersCommandHandler(
+        IOrdersRepository ordersRepository, 
+        IBusinessesRepository businessRepository,
+        IUsersRepository usersRepository,
+        IHttpContentResolver httpContentResolver,
+        IOrderModelsResolver modelsResolver)
+    {
+        _ordersRepository = ordersRepository;
+        _businessRepository = businessRepository;
+        _usersRepository = usersRepository;
+        _httpContentResolver = httpContentResolver;
+        _modelsResolver = modelsResolver;
+    }
+
+    public async Task<IActionResult> Handle(CreateOrdersCommand command, CancellationToken cancellationToken)
+    {
+        var orderRequest = await _httpContentResolver.GetPayloadAsync<CreateOrderRequest>(command.Request);
+
+        var businessDao = await _businessRepository.GetAsync(Guid.Parse(orderRequest.BusinessId));
+        var userDao = await _usersRepository.GetAsync(Guid.Parse(orderRequest.UserId));
```

Comment:
await user and business at the same time `await Task.WhenAll()`

Reply:
done